pkg/appdefinition/acornfile-schema.acorn

```diff
@@ -215,12 +215,15 @@
 		match "probes|probe":                     Probes
 		match "depends[oO]n|depends_on|consumes": string || [string]
 		match "mem|memory":                       int
+		user?:                                    User
 		permissions?: {
 			rules: [RuleSpec]
 			clusterRules?: [ClusterRuleSpec]
 		}
 	}
 
+	User: string =~ "^[0-9]+(:[0-9]+)?$"
```

Comment:
Can we have this match support int also?  I'm going to be annoying if I type `user: 1000` and it blows up because it needs to be `user: "1000"`.  Also I assume you support UID only?  Like GID is not always required?

Reply:
Yeah that makes sense.
GID defaults to UID if only one is given.

Reply:
Done